lib/alchemy/spree/engine.rb

```diff
@@ -8,8 +8,9 @@ class Engine < ::Rails::Engine
       engine_name 'alchemy_spree'
 
       initializer 'spree.user_class', after: 'alchemy.include_authentication_helpers' do
-        ::Spree.user_class = "Alchemy::User"
-        require File.join(File.dirname(__FILE__), '../../spree/authentication_helpers')
+        if ::Spree.user_class = "Alchemy::User"
+          require File.join(File.dirname(__FILE__), '../../spree/authentication_helpers')
+        end
```

Comment:
This should be generated as an initializer for the `Alchemy::User` generator option.